packages/runtime/container-runtime/package.json

```diff
@@ -114,6 +114,13 @@
 		"previousVersionStyle": "^previousMajor",
 		"baselineRange": ">=2.0.0-internal.3.0.0 <2.0.0-internal.4.0.0",
 		"baselineVersion": "2.0.0-internal.3.0.0",
-		"broken": {}
+		"broken": {
+			"EnumDeclaration_RuntimeHeaders": {
+				"forwardCompat": false
+			},
```

Comment:
Added this

Reply:
As long as the build passes, this is fine.